Libraries/Opc.Ua.Server/Configuration/ConfigurationNodeManager.cs

```diff
@@ -298,6 +298,19 @@ public NamespaceMetadataState CreateNamespaceMetadataState(string namespaceUri)
         /// <exception cref="ServiceResultException"/>
         /// <seealso cref="StatusCodes.BadUserAccessDenied"/>
         public void HasApplicationSecureAdminAccess(ISystemContext context)
+        {
+            HasApplicationSecureAdminAccess(context, "");
+        }
+
+
+        /// <summary>
+        /// Determine if the impersonated user has admin access.
+        /// </summary>
+        /// <param name="context"></param>
+        /// <param name="trustedStorePath"></param>
+        /// <exception cref="ServiceResultException"/>
+        /// <seealso cref="StatusCodes.BadUserAccessDenied"/>
+        public void HasApplicationSecureAdminAccess(ISystemContext context, string trustedStorePath)
```

Comment:
is this trustedStorePath still needed?

Reply:
It s needed to Check if the Trust List belongs to the calling Application, If you can think of a cleaner way it woudl be highly apreciated.

Reply:
the parameter is just not used in this function, thats why I am asking.